Brisk/Controllers/AuthenticationViewController.swift

```diff
@@ -22,7 +22,7 @@ final class AuthenticationViewController: ViewController {
     // MARK: - Private Methods
 
     @IBAction private func login(_ sender: Any) {
-        let username = self.appleIDTextField.stringValue
+        let username = self.appleIDTextField.stringValue.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Ah nice, can you do this for the password field as well I guess

Reply:
Doing this on the password field seems a bit too heavy-handed since anyone with a leading or trailing space in a password couldn't use the app anymore.

Reply:
Right. I don't know Apple's policy on password but I have seen other apps that accept leading trailing whitespace as a valid password. Leading and trailing whitespace for emails though is invalid.

Reply:
sg, no need to do that for now